src/vscode-dts/vscode.proposed.getSessions.d.ts

```diff
@@ -22,4 +26,33 @@ declare module 'vscode' {
 		 */
 		export function getSessions(providerId: string, scopes: readonly string[]): Thenable<readonly AuthenticationSession[]>;
 	}
+
+	/**
+	 * The options passed in to the provider when creating a session.
+	 */
+	export interface AuthenticationProviderCreateSessionOptions {
```

Comment:
This is a total nit, but the title of this file `getSessions` could be specific about what kind of session it refers to, e.g. `getAuthenticationSessions` or `authenticationGetSessions`.